builtins/amp-pixel.js

```diff
@@ -95,6 +107,49 @@ export class AmpPixel extends BaseElement {
   }
 }
 
+/**
+ * @param {!Element} parentElement
+ * @param {string} src
+ * @returns {!Element}
+ */
+function createNoReferrerPixel(parentElement, src) {
+  if (isReferrerPolicySupported()) {
+    return createImagePixel(src, true);
+  } else {
+    // if "referrerPolicy" is not supported, use iframe wrapper
+    // to scrub the referrer.
+    const iframe = createElementWithAttributes(
+        /** @type {!Document} */ (parentElement.ownerDocument), 'iframe', {
+          src: `javascript: '<img src="${src}">'`,
```

Comment:
Good catch. Fixed. Added a manual test.

I'm also considering changing the fixture of our integration test, so that CSP is always enforced. But I'd probably do in a separate PR.